tests/utils/exec/machine-pools.go

```diff
@@ -58,18 +59,47 @@ func (mp *MachinePoolService) Create(createArgs *MachinePoolArgs, extraArgs ...s
 	return nil
 }
 
-func (mp *MachinePoolService) Output() (mpName string, err error) {
+func (mp *MachinePoolService) Output() (MachinePoolArgs, error) {
 	mpDir := CON.MachinePoolDir
 	if mp.ManifestDir != "" {
 		mpDir = mp.ManifestDir
 	}
+	var args MachinePoolArgs
 	out, err := runTerraformOutput(context.TODO(), mpDir)
 	if err != nil {
-		return "", err
+		return args, err
+	}
+	if err != nil {
+		return args, err
+	}
+	fmt.Println(out["replicas"])
+	//mpNameObj := out["id"]
+	//mpName = h.DigString(mpNameObj, "value")
+	// MachinePoolArgs.Name = h.DigString(out["name"], "value")
+	// MachinePoolArgs.Replicas = h.DigInt(out["replicas"], "value")
+	// MachinePoolArgs.ID = h.DigString(out["machine_pool_id"], "value")
+	// MachinePoolArgs.MachineType = h.DigString(out["machine_type"], "value")
+	// MachinePoolArgs.AutoscalingEnabled = h.DigBool(out["autoscaling_enabled"])
```

Comment:
lots of commented-out code. can we clean it up?

Reply:
Done